easybuild/easyconfigs/b/BCFtools/BCFtools-1.3-foss-2016a.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'ConfigureMake'
+
+name = 'BCFtools'
+version = '1.3'
+
+homepage = 'http://www.htslib.org/'
+description = """Samtools is a suite of programs for interacting with high-throughput sequencing data.
+ BCFtools - Reading/writing BCF2/VCF/gVCF files and calling/filtering/summarising SNP and short indel sequence variants"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = [SOURCELOWER_TAR_BZ2]
+source_urls = ['https://github.com/samtools/%(namelower)s/releases/download/%(version)s']
+
+patches = ['%(name)s-%(version)s_extHTSlib_Makefile.patch']
+
+dependencies = [
+    ('HTSlib', '1.3'),
+    ('zlib', '1.2.8'),
+    ('GSL', '2.1'),
+]
+
+parallel = 1
+
+skipsteps = ['configure']
+
+installopts = ' prefix=%(installdir)s'
```

Comment:
@RvDijk compared to the existing `BCFtools-1.3-intel-2016a.eb` easyconfig you're missing this:

```python
buildopts = 'CC="$CC" CFLAGS="$CFLAGS" USE_GSL=1'
```